setup-env.sh

```diff
@@ -6,6 +6,9 @@ if [[ "$#" == 1 ]]; then
     machine="$1"
 else
     machine="$( "$parent_dir"/src/machine.sh )"
+    if [[ "$machine" == "balfrin" || "$machine" == "tasna" ]]; then
+        machine="$USER_ENV_ROOT"
```

Comment:
What sets `$USER_ENV_ROOT`?

Is this something we can control within this repo?
We need to otherwise we cannot easily test a new upstream.

Reply:
CSCS controls it and André dictates it.
We would have still been able to test any upstream with `. setup-env.sh /mch-environment/v99`, but I just realized that I would have accidentally introduced a new feature.
Good catch!